api/helpers.js

```diff
@@ -22,9 +22,10 @@ const buildBirthParams = (attrs) => _.pickBy({
   forenames: attrs.forenames,
   dateofbirth: attrs.dob && toInternationalDateFormat(attrs.dob)
 });
-const buildAuditParams = (attrs) => ({
+const buildAuditParams = (attrs) => _.pickBy({
   from: toInternationalDateFormat(attrs.from),
-  to: toInternationalDateFormat(attrs.to)
+  to: toInternationalDateFormat(attrs.to),
+  user: attrs.user
 });
```

Comment:
I suppose it's documented in the tests, so it's not so important. I'd still think it's better to be explicit though.
What are your thoughts?